job_bundles/README.md

```diff
@@ -0,0 +1,313 @@
+parameterValues:
+- name: deadline:targetTaskRunStatus
+  value: READY
+- name: deadline:maxFailedTasksCount
+  value: 10
+- name: deadline:maxRetriesPerTask
+  value: 5
+- name: deadline:priority
+  value: 75
+- name: BlenderSceneFile
+  value: /private/tmp/bundle_demo/bmw27_cpu.blend
+- name: Frames
+  value: 1-10
+- name: OutputDir
+  value: /private/tmp/bundle_demo/output
+- name: OutputPattern
+  value: output_####
+- name: Format
+  value: PNG
+- name: CondaPackages
+  value: blender
+- name: RezPackages
+  value: blender
+```
+
+We could recreate the exact same Job by submitting this new Job Bundle instead with: 
+
+```
+deadline bundle submit ~/.deadline/job_history/\(default\)/2024-06/2024-06-20-01-JobBundle-Demo/
+```
+
+Note: The submitted Job Bundle is saved to your "job history directory" and the location of that directory 
+can be found by running `deadline config get settings.job_history_dir`.
+
+### Elements - Asset References
+
+The `asset_references.yaml`/`asset_references.json` file defines the input and output files that the Job will access
+when it runs. Deadline Cloud uses this as an interface for its
+[Job Attachments feature](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/storage-job-attachments.html).
+If you do not have all of your inputs and outputs for Job Attachments listed in this file
+then you can still select them for your Job during submission with the `deadline bundle gui-submit` CLI command.
+
+The format of the file and the purpose of each element is as follows:
+
+```yaml
+assetReferences:
+    inputs:
+        # Filenames on the submitting workstation whose file contents are needed as 
+        # inputs to run the Job.
+        filenames:
+        - <list of file paths>
+        # Directories on the submitting workstation whose contents are needed as inputs
+        # to run the Job.
+        directories:
+        - <list of directory paths>
+
+    outputs:
+        # Directories on the submitting workstation where the Job would write output files
+        # if it was running locally.
+        directories:
+        - <list of directory paths>
+
+    # Paths that are referenced by the job, but not necessarily input or output.
+    # Use this if your job uses the name of a path in some way, but does not explicitly need
+    # the contents of that path.
+    referencedPaths:
+    - <list of directory paths>
+```
+
+When selecting which input or output files to upload to [Amazon S3](https://aws.amazon.com/s3/), the Job Attachments
+feature compares the file path against the paths listed in your
+[Storage Profiles](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/storage-shared.html). Each Storage Profile
+abstracts a network fileshare on your network that is mounted on your workstations and worker hosts, so Job Attachments
+only uploads files to S3 if the file is not contained on one of these file shares.
```

Comment:
Small detail to adjust here - this only applies to SHARED storage profile file system locations. Files in LOCAL file system locations will be attached to the job.